<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>multipleSendingClass</class>
 <widget class="QWidget" name="multipleSendingClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>multipleSending</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <widget class="QTextEdit" name="messageEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QTreeWidget" name="contactListWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>170</width>
       <height>0</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>170</width>
       <height>0</height>
      </size>
     </property>
     <property name="indentation">
      <number>10</number>
     </property>
     <property name="animated">
      <bool>true</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>1</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="sendButton">
     <property name="text">
      <string>Send</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/icons/crystal_project/message.png</normaloff>:/icons/crystal_project/message.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="stopButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/icons/crystal_project/stop.png</normaloff>:/icons/crystal_project/stop.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
